shared/oval/logwatch_configured_hostlimit.xml

```diff
@@ -3,10 +3,10 @@
     <metadata>
       <title>Ensure Logwatch HostLimit Configured</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
+        <platform>multi_platform_rhel</platform>
       </affected>
       <description>Test if HostLimit line in logwatch.conf is set appropriately.</description>
-      <reference source="MED" ref_id="20130819" ref_url="test_attestation" />
+      <reference source="galford" ref_id="20160120" ref_url="test_attestation" />
     </metadata>
     <criteria operator="AND">
       <criterion comment="Test value of HostLimit" test_ref="test_logwatch_configured_hostlimit" />
```

Comment:
There's an issue with the pattern match below (there's exactly one space required before ```HostLimit``` setting).

This should be enhanced via previously proposed patch (which seem to have fallen through cracks when we yet used the SSG ML for proposing the patches):

The particular proposal is this one:
  https://lists.fedorahosted.org/pipermail/scap-security-guide/2014-March/005084.html

I will create a new PR post merging this one.